.github/workflows/build.yml

```diff
@@ -25,8 +25,30 @@ jobs:
       - name: Install tox
         run: pip install tox==3.17.1
       - name: Run Unit tests
-        run: tox -e py  # Run tox using the version of Python in `PATH`
+        run: cd code42cli; tox -e py  # Run tox using the version of Python in `PATH`
       - name: Submit coverage report
         uses: codecov/codecov-action@v1.0.7
         with:
-          file: ./coverage.xml
+          file: code42cli/coverage.xml
+      - name: Checkout mock servers
+        uses: actions/checkout@v2
+        with:
+          ref: network
```

Comment:
TODO: remove this once https://github.com/code42/code42-mock-servers/pull/9 merges